library/Zend/Db/Adapter/Pdo/Sqlite.php

```diff
@@ -91,6 +91,14 @@ public function __construct(array $config = [])
         $this->_config['username'] = null;
         $this->_config['password'] = null;
 
+        if (PHP_VERSION_ID >= 80100) {
+            // ensure $config['driver_options'] is an array
+            $config['driver_options'] = empty($config['driver_options']) ? array() : $config['driver_options'];
```

Comment:
```suggestion
            $config['driver_options'] = $config['driver_options'] ?? [];
```

you want to use newer language features here.